packages/components/src/call-out/styles.scss

```diff
@@ -0,0 +1,34 @@
+@import "@wordpress/base-styles/colors";
+@import "@wordpress/base-styles/mixins";
+@import "@wordpress/base-styles/variables";
+
+.call-out {
+	max-width: 600px;
+
+	.call-out-h-container {
+		padding: $grid-unit-40;
+		@include body-medium;
+		color: $gray-700;
+
+		> * {
+			flex: 1 0;
```

Comment:
I've newish to `HStack` and `VStack`. I'm confused why there aren't props for setting the `flex-grow` and `flex-basis` of the children. Am I missing something?

Reply:
The problem is I definitely want to set `flex-basis` to `0` too. But TIL that `expanded` is there for the most usual case